src/providers/android.js

```diff
@@ -24,11 +24,12 @@ export const getAndroidVersion = async (bundleId, country) => {
 
   const text = await res.text();
   const version = text.match(/\[\[\[['"]((\d+\.)+\d+)['"]\]\],/)[1];
+  const notes = text.match(/<div itemprop="description">(.*?)<\/div>/)[1];
```

Comment:
suggestion: should we add a `?` here to make sure this doesn't error if the div can't be found?

```suggestion
  const notes = text.match(/<div itemprop="description">(.*?)<\/div>/)?.[1];
```

Reply:
In `version` this check is not necessary because version is always set. Description of provided changes is optional. You are right. Good catch

Reply:
Yes and without the version the response isn't really useful I think, whereas without notes it might still be useful.

Reply:
Readme file could be updated, currently notes property is marked as iOS only